pkg/services/publicdashboards/models/dashboard_queries.go

```diff
@@ -1,4 +1,4 @@
-package query
+package models
```

Comment:
It has a lot of logic and functions. Wouldn't be better to put in `service` package instead of `models`?

Reply:
What kind of things should we store in the models folder? Just struct types?

Reply:
I would say structs, interfaces, some getters/setters or small functions related to those structs. These functions seems more related to the service than the models

Reply:
Sounds good. Wasn't sure what our standard was. I'll move these into the service package